### DISCUSSION TITLE
Notifications Limit

### DISCUSSION THREAD

**Discussion by KSumwalt** [andrei-tatar/node-red-contrib-smartnora#210] [General]
Is there a limitation to the number of notifications one can receive in a day?  Or maybe there is an issue with them being sent at the moment?  I was testing a new flow and notification message and am no longer receiving any notifications.  I did try re-subscribing the computer but not my phone.

Edit:  Adding that I even imported the sample flow for the Notification node and it is not working while it had minutes before I stopped receiving notifications.

**Comment by KSumwalt** [comment]
Seems to have resolved itself.

**Reply by andrei-tatar** [reply]
@KSumwalt there is no limit, however I did notice there is sometimes a delay until the notification gets through. 
I suspect that mobile devices go to sleep and check for notifications on an interval (every few minutes or something)
On desktop, not sure. But I did see different behavior between chrome on win10 and win11
